packages/design-system-web/.storybook/preview.js

```diff
@@ -12,7 +12,7 @@ export const parameters = {
         'Welcome',
         'Getting Started',
         'Design', ['Overview', 'Layout', 'Typography', 'Theming', 'Theme Generator', 'Theme Generator Color'],
-        'Contributing'
+        'Contributing', ['CI And CD Workflow', 'Tutorial', ['Introduction', 'Setup', 'Create Component', 'Data Fetching', 'Component Props', 'Create Data Visualization Component', 'Server Side Rendering']],
```

Comment:
I'm changing the order here to put SSR last, since it has nothing to do with the rest.

Reply:
perfect, thank you.